### PR TITLE
MNT Use GitHub Actions CI

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/gha-ci/issues/11<br /><br />Workflow run https://github.com/creative-commoners/silverstripe-ldap/actions?query=branch%3Apulls%2F1.3%2Fmodule-standards

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
         "php": "^7.3 || ^8.0",
         "silverstripe/framework": "^4.10",
         "silverstripe/cms": "^4.1",
-        "symbiote/silverstripe-queuedjobs": "^4",
+        "symbiote/silverstripe-queuedjobs": "^4.9",
```
Comment (ChloeSartorelli) [inline composer.json]: Why is this increased?

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (emteknetnz) [reply composer.json]: Fix prefer-lowest build
https://github.com/creative-commoners/silverstripe-ldap/runs/7191398040?check_suite_focus=true#step:10:687
```
ERROR [Emergency]: Uncaught Error: Call to undefined method Symbiote\QueuedJobs\Services\QueuedJobService::singleton()
[2022-07-05 07:02:02] error-log.ERROR: Uncaught Exception Error: "Call to undefined method Symbiote\QueuedJobs\Services\QueuedJobService::singleton()" at /home/runner/work/silverstripe-ldap/silverstripe-ldap/vendor/silverstripe/session-manager/src/Jobs/GarbageCollectionJob.php line 65 {"exception":"[object] (Error: Error: Call to undefined method Symbiote\\QueuedJobs\\Services\\QueuedJobService::singleton() 
```

Comment (ChloeSartorelli) [reply composer.json]: `QueuedJobService` is `Injectable` as of `4.1.0`.
Please use `^4.1` as the constraint.

Comment (emteknetnz) [reply composer.json]: As [noted](https://github.com/silverstripe/silverstripe-session-manager/pull/112#discussion_r922905438) - phpunit 9 compatibility

**Review by GuySartorelli — APPROVED** [review]
CI run on creative-commoners is green for this PR.

**Pull request merged**